_build_cfg.yml

```diff
@@ -333,6 +333,20 @@ Topics:
     Distros: openshift-enterprise
 
 ---
+Name: Administrator Solutions
+Dir: admin_solutions
+Distros: openshift-origin,openshift-enterprise,atomic-*
```

Comment:
@tpoitras With `atomic-*` here, this will end up getting built into the nightly build at http://docs.projectatomic.io/registry for Atomic Registry. Is that intentional?